engine.py

```diff
@@ -1638,9 +1638,9 @@ def __on_worker_failure(self, uid, msg):
                 There was an error retrieving fields for this context. Please
                 see the logs for the specific error message. If this is a
                 recurring error and you need further assistance, please
-                send an email to <a href='mailto:support@shotgunsoftware.com'>
-                support@shotgunsoftware.com</a>.
-                """
+                contact our support team via {}""".format(
```

Comment:
```suggestion
                contact our support team via {}.""".format(
```
I think you've lost the full stop from the sentence.